module1/homework/grayscaleImages/compression.hpp

```diff
@@ -0,0 +1,24 @@
+// #pragma once
+
+// #include <array>
+// #include <vector>
+// #include <utility>
+
+// constexpr size_t width = 240;
+// constexpr size_t height =160; 
+
+// std::vector<std::pair<uint8_t, uint8_t>> compressGrayscale(const std::array<std::array<uint8_t, width>, height>& image);
+// std::array<std::array<uint8_t, width>, height> decompressGrayscale(std::vector<std::pair<uint8_t, uint8_t>> comressedImage);
+
+
+#pragma once
+
+#include <array>
+#include <vector>
+
+constexpr size_t width = 32;
+constexpr size_t height = 32;
+
+std::vector<std::pair<uint8_t, uint8_t>> compressGrayscale(const std::array<std::array<uint8_t, width>, height>& arr);
+std::array<std::array<uint8_t, width>, height> decompressGrayscale(std::vector<std::pair<uint8_t, uint8_t>>);
+// void printMap(std::array<std::array<uint8_t, width>, height>& bitmap);
```

Comment:
Did you write printMap function? If no this comment can be deleted

Reply:
+1